packages/prerender-fargate/lib/prerender-fargate.ts

```diff
@@ -114,5 +115,16 @@ export class PrerenderFargate extends Construct {
             scaleInCooldown: Duration.seconds(60),
             scaleOutCooldown: Duration.seconds(60),
         });
+
+        /**
+         * Enable VPC Endpoints for S3
+         * This would  create S3 endpoints in all the PUBLIC subnets of the VPC
+         */
+        if (props.enableS3Endpoint) {
```

Comment:
Should this default to true?

Reply:
This would be associated with a cost so it should be optional.

Reply:
But doesn't it make it cheaper? Surely that should be a default

Reply:
As long as it's backward-compatible, which it is AFAIK, I'm happy with making this default, too.

Reply:
I'm happy to make this as default, but this would NOT make anything cheaper. You would have a running cost (per GB) and cost of having the endpoint (~0.01 USD X number of subnets).